jcl/src/java.base/share/classes/java/lang/Class.java

```diff
@@ -1303,11 +1315,24 @@ public Field getField(String name) throws NoSuchFieldException, SecurityExceptio
  * @return		Class<?>[]
  *					the interfaces the receiver claims to implement.
  */
-public Class<?>[] getInterfaces()
-{
-	return J9VMInternals.getInterfaces(this);
+public Class<?>[] getInterfaces(){
+	if (cachedInterfaces == null){
+		if (cachedInterfacesOffset == -1) {
+			Field cachedInterfacesField;
```

Comment:
The field is only used inside the `try` block.  It can be declared inside.

Reply:
Thanks for the comments! Modified accordingly.